blueocean-rest/README.md

```diff
@@ -529,7 +535,9 @@ Each branch in the repo with Jenkins file will appear as a branch in this pipeli
         "numberOfSuccessfulBranches": 0,
         "numberOfSuccessfulPullRequests": 0,
         "totalNumberOfBranches": 3,
-        "totalNumberOfPullRequests": 0
+        "totalNumberOfPullRequests": 0,
+        "numberOfQueuedPipelines" : 0,
+        "numberOfRunningPipelines" : 2
```

Comment:
Is there a typo here .... shouldn't `totalNumberOfPullRequests` be `2` ? (`numberOfQueuedPipelines + numberOfRunningPipelines `)